src/napari_workshop_plugin/_widget.py

```diff
@@ -51,6 +51,20 @@ def smooth_labels(
     return filters.rank.mean(labels, morphology.disk(disk_size))
 
 
+def display_layer(viewer: "napari.Viewer", args) -> None:
+    print(args)
```

Comment:
Can you either dump this line, or replace by a debug log? Maybe a debug log could be good
```suggestion
    logging.debug(args)
```